src/App.js

```diff
@@ -1,11 +1,24 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Nav from './components/Navbar/Nav';
+import Home from './components/Navbar/Home';
+import Quote from './components/Navbar/Quote';
 
 function App() {
   return (
-    <div className="App">
-      <Calculator />
-    </div>
+    <Router>
+      <div className="App">
+        <Nav />
+        <Routes>
+          <Route path="/home" exact element={<Home />} />
```

Comment:
- [OPTIONAL] It will be nice to make the home page the default page, so that you will not land on an empty page when you follow the root `path='/'`. Something like so: 
```
<Route path="/" exact element={<Home />} />
```
Also make sure you update your `Nav.js` file appropriately.